test/inductor/test_b2b_gemm.py

```diff
@@ -0,0 +1,42 @@
+# Owner(s): ["module: inductor"]
+import torch
+from torch._inductor.test_case import run_tests, TestCase
+from torch._inductor.utils import run_and_get_code
+from torch.testing._internal.inductor_utils import HAS_CUDA
+
+
+class B2BGEMMTest(TestCase):
+    @torch._inductor.config.patch(b2b_gemm_pass=True)
+    def test_b2b_gemm_supported_block_size(self):
+        """sizes match the supported block sizes"""
+
+        def f(m1: torch.Tensor, m2: torch.Tensor, m3: torch.Tensor) -> torch.Tensor:
+            return torch.matmul(torch.matmul(m1, m2), m3)
+
+        f_opt = torch.compile(f)
+        A = torch.randn((512, 64), device="cuda", dtype=torch.float16)
```

Comment:
FYI if it's useful to test for other dtypes, we have this handy decorator that makes it easy to parameterize the dtype and run multiple tests. Here's an example: https://github.com/pytorch/pytorch/blob/90f6043368c59a4fbfbf1e0e4c3f347c5b3d6ef4/test/inductor/test_codecache.py#L114